packages/veritone-react-common/src/helpers/withVeritoneSDKThemeProvider.js

```diff
@@ -27,27 +28,32 @@ export const defaultVSDKTheme = {
   }
 };
 
+const jss = create({
+  ...jssPreset()
+});
+
+const generateClassName = createGenerateClassName({
```

Comment:
This should be passed by client, If client doesn't pass one, we can create it.

https://material-ui.com/getting-started/faq/#why-arent-my-components-rendering-correctly-in-production-builds

Reply:
Done! passing via props